java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Order/AppetizerOrdered.java

```diff
@@ -0,0 +1,13 @@
+package com.my.FoodTruckApp2.Order;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@Data
+@NoArgsConstructor
+public class AppetizerOrdered {
+    private Integer orderId;
+    private Integer appetizerId;
+}
```

Comment:
this doesn't seem to be getting used at all right now.

Reply:
I will delete it